frontend/fingertips-frontend/components/molecules/Inequalities/InequalitiesTrend/inequalitiesTrend.test.tsx

```diff
@@ -30,6 +94,14 @@ describe('InequalitiesTrend suite', () => {
     expect(
       screen.getByText(/Inequalities data over time/i)
     ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('inequalitiesTypes-dropDown-component')
+    ).toBeInTheDocument();
+    expect(inequalitiesTypesDropDown).toBeInTheDocument();
+    expect(inequalitiesOptions).toHaveLength(2);
```

Comment:
Is this meant to be?:
```suggestion
    expect(inequalitiesDropDownOptions).toHaveLength(2);
```